### DISCUSSION TITLE
Want to visualize increase count of issues per month (e.g. Insights → Pulse)

### DISCUSSION THREAD

**Discussion by kenjiuno** [community/community#126088] [Repositories]
### Select Topic Area

Product Feedback

### Body

I want to confirm the increasing pace of issues of repositories about OSS that I use.

Insights → Pulse is a good way to achieve this.
But about this period selection, the longest one is _1 month_.
It is short to observe the tendency.

It will be more useful if I can
- See this in a chart format
- Select longer period

**Comment by acasademont** [comment]
Yes please, it would be great to select custom dates in the Pulse section
